server/src/test/scala/org/http4s/server/middleware/DateSuite.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.server.middleware
+
+import cats.data.OptionT
+import cats.syntax.all._
+import cats.effect._
+import org.http4s._
+import org.http4s.syntax.kleisli._
+import org.http4s.headers.{Date => HDate}
+
+class DateSuite extends Http4sSuite {
+  implicit val timer: Timer[IO] = Http4sSuite.TestTimer
```

Comment:
targeting 0.22 due to the TestTimer which doesn't seem available in 0.21

Reply:
I did the Specs2 quarantine on series/0.22 because we had some deprecated things in testing that would break bincompat to get rid of.  Part of that work was to copy over some of those vals like `TestTimer`, which we probably could have done on 0.21.

Reply:
based it on top of the test-timer branch